docs/adguard-for-windows/overview.md

```diff
@@ -3,115 +3,94 @@ title: Features overview
+![Ad Blocker]()
 
-AdGuard provides users with numerous filters that let you fine-tune blocking properties.
+In the Ad Blocker module you can:
+* enable an auto activation of language-specific filters
+* disable search ads and self-promotion blocking
+* activate pre-installed filters such as the *AdGuard Base filter*, 
+* install filters from the list or add a custom filter by clicking the *Add a filter* button, 
+* make changes to existing rule groups using the *Filter editor* button, 
+* compose a filter from your own rules written manually, imported or created with the AdGuard Assistant extension.
 
-Filters receive automatic updates from AdGuard servers. Our specialists create them all manually based on user data received. You can always help other AdGuard users by sending us website URLs that you have still seen ads at.
+Before you start manually writing your own rules read our detailed [syntax guide](/general/ad-filtering/create-own-filters.md). 
 
-## Browsing Security
+### Stealth Mode
 
-Phishing is a very common web fraud aimed at stealing your personal data (passwords, bank account details, etc.). It is carried out using fake web pages mimicking popular websites or real services, such as web stores or social networks. Many phishing websites have URLs that only differ from the real ones in just a few symbols to put off your guard. If a user enters their details on such a website, they will definitely end up in the hands of criminals.
+Many websites gather information about their visitors, such as their IP addresses, information about the browser and operating system installed, screen resolution, and even what page the user came or was redirected from. Some web pages use cookies to mark the browser and save your personal settings, user preferences, or “recognize” you upon your next visit. Stealth Mode safeguards your personal information from such data and statistics gathering systems.
 
-Browsing Security gives strong protection against such incidents. In addition, the regularly updated module bases allow it to recognize malicious websites, thus securing your PC from viruses and trojans. At the same time, the user’s personal information is not transferred anywhere, and AdGuard servers do not know which websites the user is visiting, because security check is not performed via open web address (URL) but through hash prefixes (hash is a certain data structure that renders each address added to the base unambiguous).
+![Stealth Mode]()
+ 
+You can flexibly adjust the work of Stealth Mode: for instance, you can prohibit the website to receive the search request you used to find it on the Internet, automatically delete both third-party and website’s own cookies, and disable location sharing in your browser that can be used to track your whereabouts.
+ 
+To learn all the main features and options you can manage in the Stealth mode settings, [read this article](/general/stealth-mode.md).  
```

Comment:
`To learn everything about Stealth Mode and its many options, [read this article](/general/stealth-mode.md).`